### PR TITLE
Update amazon.txt

### PR DESCRIPTION
**Pull request by gallegonovato**

### CODE REVIEW

**Comment by gallegonovato** [comment]
Good afternoon @nickspaargaren

I found these lists https://gitlab.com/Cybo1927/Hosts/-/raw/master/Anti-corp/Amazon ( https://gitlab.com/Cybo1927/Hosts/-/tree/master/Anti-corp )

I don't know what you think to add.

Best regards

File: amazon.txt [hunk 1]
```diff
@@ -2287,8 +2333,21 @@ api.twitchsvc.net
...
 appeals-backend.a.twitchcdn.net
+countess-prod-public-176850629.us-west-2.elb.amazonaws.com
 badges.twitch.tv
 blog.twitch.tv
+ext-twitch.tv
+jtvnw.net
+live-video.net
+ttvnw.net
+twitch.okta.com
+twitch.map.fastly.net
```
Comment (nickspaargaren) [inline amazon.txt:2344]:
```suggestion
```

File: amazon.txt [hunk 2]
```diff
@@ -2856,3 +2856,131 @@ dzupi9b81okew.cloudfront.net
...
+olga.ns.cloudflare.com
+pam.ns.cloudflare.com
+rob.ns.cloudflare.com
+sara.ns.cloudflare.com
+support.cloudflare.com
+uma.ns.cloudflare.com
+whois.cloudflare.com
+will.ns.cloudflare.com
+www.cloudflare.com
+zara.ns.cloudflare.com
```
Comment (nickspaargaren) [inline amazon.txt:2985]:
```suggestion
```
Cloudflare domains are not Amazons

File: amazon.txt [hunk 3]
```diff
@@ -2856,3 +2856,78 @@ dzupi9b81okew.cloudfront.net
...
+bv-analytics-js-prod.s3.amazonaws.com
+bv-analytics-js-qa.s3.amazonaws.com
+prod.amcs-tachyon.com
+web.diagnostic.networking.aws.dev
+countess-prod-public-176850629.us-west-2.elb.amazonaws.com
+twitch.okta.com
+twitchsvc.net
+ink1001.com
+outwitch.com
+twitchguru.us.intellitxt.com
```
Comment (nickspaargaren) [inline amazon.txt:2911]:
```suggestion
```

File: amazon.txt [hunk 4]
```diff
@@ -2856,3 +2856,78 @@ dzupi9b81okew.cloudfront.net
...
+newsharecounts.s3-us-west-2.amazonaws.com
+atv-ext.amazon.com
+atv-ext-fe.amazon.com
+pins.amazon.com
+pindorama.amazon.com
+guipitan.amazon.com
+cdn-simpli-fi.s3-accelerate.amazonaws.com
+atv-ps.amazon.com
+pv-cdn.net
+aiv-cdn.net.c.footprint.net
```
Comment (nickspaargaren) [inline amazon.txt:2897]:
```suggestion
```

**Pull request merged**